app/controllers/employee_controller.rb

```diff
@@ -0,0 +1,39 @@
+class EmployeeController < ApplicationController
+  protect_from_forgery with: :exception
+
+  before_action :get_employee, only:[:show, :edit, :update, :destroy]
+
+  def get_employee
+    @employee = Employee.find(params[:id])
+  end
+
+  def index
+    @employees = Employee.order(:first_name)  
+  end
+
+  def create
+    @employee = Employee.new(employee_params(params))
+    @employee.save
+
+    redirect_to employee_path(@employee)
+  end
+
+  def update
+    @employee.update_attributes(employee_params(params))
+
+    redirect_to @employee
+  end
+
+  def destroy
+    @employee.destroy
+    flash[:success] = "Employee Deleted"
+
+    redirect_to employee_index_path
```

Comment:
Since you're in the `EmployeeController`, you have the option of doing `redirect_to(:index)`. I like that option here because it means I don't have to think about where the path helper is going. I know I just have to scroll up to see where it goes.